### ISSUE TITLE
Latest release has a __MACOSX directory in zip file

### ISSUE THREAD

**Issue by micaherne** [h5p/moodle-mod_hvp#537]
The zip file of the latest release downloadable from the Moodle plugin directory (2023122500) has a __MACOSX directory at top level in addition to the hvp directory.

This causes issues with Moodle, in particular with the unit test component_test::test_get_plugin_list, which fails.

**Comment by abautu** [comment]
It also breaks composer (I use composer to manage Moodle dependencies). Due to the __MACOSX directory, the plugin is installed into moodle/mod/hvp/hvp/ (instead of moodle/mod/hvp/).

**Comment by mackensen** [comment]
I also use composer to manage my Moodle installation and this issue makes this version of the plugin uninstallable. Same issue as #531.

**Comment by otacke** [comment]
@micaherne I assume this issue could be closed now? Can't find any traces of MacOS folders in the latest version.

**Comment by micaherne** [comment]
Yes if they're not there in recent versions it's fine to close it!

**Comment by otacke** [comment]
@micaherne 👍
